packages/application-shell/src/components/public-page-container/public-page-container.js

```diff
@@ -1,31 +1,96 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
 import { FormattedMessage } from 'react-intl';
-import { Spacings, Text } from '@commercetools-frontend/ui-kit';
+import {
+  Text,
+  Spacings,
+  customProperties,
+} from '@commercetools-frontend/ui-kit';
+import LogoBWWhiteSVG from '@commercetools-frontend/assets/images/ct_logo_bw_white.svg';
+import BackgroundImage from '@commercetools-frontend/assets/images/public-background.png';
 import PortalsContainer from '../portals-container';
-import styles from './public-page-container.mod.css';
 import messages from './messages';
 
 const year = new Date().getUTCFullYear();
 
-const PublicPageContainer = props => (
-  <React.Fragment>
-    <PortalsContainer />
-    <div className={styles.container}>
-      <Spacings.Stack scale="m">
-        {props.children}
-        <Spacings.Inline justifyContent="space-between">
-          <a href={`https://commercetools.com/privacy`} target="_blank">
-            <Text.Detail>
-              <FormattedMessage {...messages.privacyPolicy} />
-            </Text.Detail>
-          </a>
-          <Text.Detail>{`${year} © commercetools`}</Text.Detail>
-        </Spacings.Inline>
-      </Spacings.Stack>
-    </div>
-  </React.Fragment>
-);
+const PublicPageContainer = props => {
+  const privacyPolicyLink = (
+    <a href={`https://commercetools.com/privacy`} target="_blank">
+      <FormattedMessage {...messages.privacyPolicyLink} />
+    </a>
+  );
+
+  const termsOfServiceLink = (
+    <a href={`https://commercetools.com/terms`} target="_blank">
+      <FormattedMessage {...messages.termsOfServiceLink} />
+    </a>
+  );
+
+  return (
+    <React.Fragment>
+      <PortalsContainer />
+      <div
+        css={css`
+          height: 100vh;
+          display: flex;
+          padding-top: ${customProperties.spacing32};
+          justify-content: center;
+          background-color: #073538;
+          background-image: url('${BackgroundImage}');
+        `}
+      >
+        <Spacings.Stack scale="xl" alignItems="center">
+          <Spacings.Stack scale="m" alignItems="center">
+            <img
+              css={css`
+                width: 275px;
+                height: 45px;
+              `}
+              src={LogoBWWhiteSVG}
+            />
+            <div>
+              <Text.Headline elementType="h2">
+                <span style={{ color: 'white' }}>
+                  <FormattedMessage {...messages.welcome} />
+                </span>
+              </Text.Headline>
+            </div>
+          </Spacings.Stack>
+          <Spacings.Stack scale="s">
+            {/* Login Card */}
+            {props.children}
+            <div
```

Comment:
Can you explain why this wrapper is needed and Spacings.Stack can't be used with the same props?

Reply:
Now that I took another look, I think this is a leftover 😅